client/my-sites/plans/jetpack-plans/product-lightbox/style.scss

```diff
@@ -315,6 +315,36 @@
 	}
 }
 
+.product-lightbox__variants-plan-card-price,
+.product-lightbox__variants-plan-card-old-price {
+	h4 {
+		display: inherit;
+	}
+	.plan-price__fraction {
+		font-weight: inherit;
+		font-size: inherit;
+		vertical-align: inherit;
+	}
+	.plan-price__integer {
+		margin: 0;
+		font-weight: inherit;
+	}
+	.plan-price__currency-symbol {
+		color: inherit;
+		font-size: inherit;
+		vertical-align: inherit;
+	}
+	.plan-price.is-original::before {
+		content: none;
+	}
+	.plan-price {
+		margin: 0;
```

Comment:
Just a nit-pick, but on the main pricing page, in the simple product cards, there is a slight space between the price number and the timeframe text: (See image)

![Markup 2022-09-22 at 18 53 19](https://user-images.githubusercontent.com/11078128/191865191-e087c887-4d66-4505-ae33-db9874f5cdd7.png)

I wonder if we should also have a slight space in the lightbox too? Maybe like this?

```suggestion
		margin: 0 0.2rem 0 0;
```

.
BEFORE | AFTER
--- | ---
![Screenshot on 2022-09-22 at 18-57-33](https://user-images.githubusercontent.com/11078128/191865843-214d5851-2654-43ab-90c0-685982f8a7ee.png) |  ![Screenshot on 2022-09-22 at 18-58-17](https://user-images.githubusercontent.com/11078128/191865869-5434443a-e715-48d8-8fc7-70209704367c.png)